docker/almalinux-8-amd64-openssl-1.1.x/run.sh

```diff
@@ -30,3 +30,4 @@ docker run -p 3000:3000 \
 sleep 15
 
 pnpm install
+pnpm prisma generate
```

Comment:
I rather add it here instead of having a custom project exclusion.

Reply:
Why is all this needed? This should happen automatically, under the hood, right?

Reply:
Because our dependencies are cached to speed up the runs, we end up in a situation where the `postinstall` of Prisma Client is not always triggered, similar to what we had with Vercel.